zadaci/sa-kolokvijuma/2020/PSI/T5/G1/zadatak.rst

```diff
@@ -0,0 +1,38 @@
+Iz zadate ulazne datoteke učitati podatke u jednostruko spregnutu listu, gde
+struktura ``knjiga_st`` sadrži sledeća polja:
+
+- ime pisca (jedna reč, do 20 karaktera)
+- prezime pisca (jedna reč, do 20 karaktera)
+- naslov knjige (jedna reč, do 30 karaktera)
+- cena po komadu (pozitivan broj)
+- broj prodatih kopija (pozitivan broj)
+
+Naravno, struktura ``knjiga_st`` sadrži i polja potrebna za pravilno
+formiranje jednostruko spregnute liste.
+
+Učitati knjige u jednostruko spregnutu listu i uraditi sledeće:
+
+#. Sortirati knjige po ceni u rastućem redosledu pri dodavanju u listu
+#. Izračunati ukupnu zaradu svake knjige korišćenjem cene po komadu i broja prodatih kopija.
+#. Na standardni izlaz (ekran Terminala) ispisati sledeće informacije:
+    - prezime pisca
+    - ime pisca
+    - naslov knjige
+    - ukupna zarada
+#. Pronaći knjigu koja je zaradila najviše novca i ispisati njene podatke na standardni izlaz (ekran Terminala)
```

Comment:
There has to be one blank line between numbered and nested list (from both sides) or it won't render properly. Please, take a look how 3. is **bold** because of this in the PDF version:
![image](https://user-images.githubusercontent.com/12072588/117368116-60c04080-aec3-11eb-8a9a-78733f2ba282.png)